src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -54,13 +54,13 @@ class ResourceType(Enum):  # pylint: disable=too-few-public-methods
     MGMT_APPSERVICE = ('azure.mgmt.web', 'WebSiteManagementClient')
     MGMT_IOTHUB = ('azure.mgmt.iothub', 'IotHubClient')
     MGMT_ARO = ('azure.mgmt.redhatopenshift', 'AzureRedHatOpenShiftClient')
+    MGMT_CONTAINERSERVICE = ('azure.mgmt.containerservice', 'ContainerServiceClient')
```

Comment:
As discussed can we add hybrid profile for CONTAINERREGISTRY version 2019-07-01 as well.